src/seedu/addressbook/data/menu/ReadOnlyMenus.java

```diff
@@ -37,19 +37,32 @@ default String getAsTextShowAll() {
         builder.append(getName())
                 .append(" Price: ");
         builder.append(getPrice())
+                .append(" Type: ");
+        builder.append(getType())
                 .append(" Tags: ");
         for (Tag tag : getTags()) {
             builder.append(tag);
         }
         return builder.toString();
     }
 
+    /**
+     * Formats the menu name and price as text, showing all relevant details.
+     */
+    default String getAsTextShowMenuAndPrice() {
+        final StringBuilder builder = new StringBuilder();
+        builder.append(getName())
+                .append(" Price: ");
+        builder.append(getPrice());
+        return builder.toString();
+    }
+
     /**
      * Formats a menu item as text, showing only non-private contact details.
      */
     default String getAsTextHidePrivate() {
```

Comment:
Should merge `getAsTextShowAll()` and `getAsTextShowPrivate()` into one single method `getAsText()`